MANIFEST.in

```diff
@@ -1,2 +1,4 @@
 include versioneer.py
 include nanshe_workflow/_version.py
+
+include nanshe_workflow/dask_config.yaml
```

Comment:
Should already be handled by `package_data`, but this is a secondary measure to ensure the configuration file is included.